src/unix/linux_like/linux/arch/mips/mod.rs

```diff
@@ -383,6 +383,12 @@ cfg_if! {
         linux_time_bits64
     ))] {
         pub const RLIM_INFINITY: crate::rlim_t = !0;
+    } else if #[cfg(all(
+        any(target_arch = "mips", target_arch = "mips32r6"),
+        target_env = "gnu",
+        gnu_file_offset_bits64
+    ))] {
+        pub const RLIM_INFINITY: crate::rlim_t = !0;
```

Comment:
Is this meant to be the same as the above block? If so, the conditions could be merged.

Reply:
The conditions tend to be complicated, but I'll push an updated variant.